projects/js-packages/publicize-components/src/types/types.ts

```diff
@@ -1,3 +1,12 @@
+export interface FeatureFlags {
+	useAdminUiV1: boolean;
```

Comment:
Do we have to add the new feature flag for the modal as well?

Reply:
Yes, we need to migrate all our initial state to this new consolidated one.